src/github.com/getlantern/proxiedsites/proxiedsites.go

```diff
@@ -1,269 +1,217 @@
-// package proxiedsites is a module used to manage the list of sites
-// being proxied by Lantern
-// when the list is modified using the Lantern UI, it propagates
-// to the default YAML and PAC file configurations
+// proxiedsites manages a list of proxied sites, including a default set of
+// sites (cloud) and user-applied customizations to that list. It provides an
+// implementation of the http.Handler interface that serves up a PAC file based
+// on the currently active proxied sites (cloud + additions - deletions).
 package proxiedsites
```

Comment:
Where is the flashlight.yaml file now? Is that no longer being generated? I use to be able to update the list of proxied sites directly from there. How do I make on-the-fly changes to the list now without going through the UI?